indico/modules/events/persons/templates/email_dialog.html

```diff
@@ -0,0 +1,17 @@
+{% from 'forms/_form.html' import simple_form %}
+{{ simple_form(form, submit=_('Send')) }}
+<script>
+    var emailCountLimit = 10;
```

Comment:
I think just using `10` in the `if` below is still clear. It's used in exactly one place and it's extremely obvious what it's used for